src/istgt_lu_ctl.c

```diff
@@ -2959,17 +2960,31 @@ istgt_uctl_cmd_iostats(UCTL_Ptr uctl)
 		size = malloc(length + 1);
 		snprintf(size, length + 1, "%"PRIu64, spec->size);
 
+        usedlogicalblock = ( spec->stats.used / spec->blocklen ) ;
+
+        length = snprintf(NULL, 0, "%"PRIu64, usedlogicalblock);
```

Comment:
this idea of 'snprintf' with 0 looks good to get the length of number after printing as string.